draft-hoffman-dns-over-https-latest.mkd

```diff
@@ -175,49 +181,123 @@ the well-known path allows automated discovery of a DNS API Service,
 and also helps contextualize DNS Query requests pushed over an active
 HTTP/2 connection.
 
-Some forms of the request may also include a HTTP query as defined by
-{{RFC3986}}.
-
-A DNS API Client encodes the DNS query into the HTTP request in one
-of two different methods:
+A DNS API Client encodes the DNS query into the HTTP request
+using the HTTP GET or POST methods.
 
 GET:
-: Using the on-the-wire representation of a DNS message defined in
-{{RFC1035}} as the query string portion of the URI, encoded with base64url {{RFC4648}},
-using the GET HTTP method. Padding characters for base64url MUST NOT be included. This is the
-preferred approach because it is friendlier to HTTP caches.
+: Using the on-the-wire representation of a DNS message defined in {{dnswire}}
+or the simple JSON format defined in {{simplejson}}
+as the query string portion of the URI.
 
 POST:
 : Including the DNS query as the message body of the HTTP
 request, with the request using the POST method. The body MUST use a media
-type selected by the DNS API Client, and that media type MUST be indicated by the
+type selected by the DNS API client, and that media type MUST be indicated by the
 request's Content-Type header.
 
-The DNS request MAY have one or more EDNS(0) extensions {{RFC6891}}.
+Using GET is the preferred approach because it is friendlier to HTTP caches.
+
+When using the GET method, the URI path MUST contain
+?content-type=TTT&body=BBBB, where "TTT" is the media type of
+the format used for the request, and "BBB" is the content of
+the body, as described in {{dnswire}} and {{simplejson}}.
 
 The DNS API Client SHOULD include an HTTP "Accept:" request header to
 say what type of content can be understood in response. The client
-MUST be prepared to process "application/dns-udpwireformat" responses
-but MAY process any other type it receives.
+MUST be prepared to process "application/dns-udpwireformat" 
+and "application/simpledns+json"
```

Comment:
While I'd like to see json flourish, I'm not sure we have the right format yet and its certainly not the LCD. why can't this just stay with a single MTI?